mdm/common/src/main/java/com/gentics/mesh/core/data/dao/PersistingLanguageDao.java

```diff
@@ -26,11 +30,38 @@ default HibLanguage create(String languageName, String languageTag, String uuid)
 	@Override
 	default LanguageResponse transformToRestSync(HibLanguage element, InternalActionContext ac, int level,
 			String... languageTags) {
+		GenericParameters generic = ac.getGenericParameters();
+		FieldsSet fields = generic.getFields();
+
 		LanguageResponse model = new LanguageResponse();
-		model.setUuid(element.getUuid());
-		model.setLanguageTag(element.getLanguageTag());
-		model.setName(element.getName());
-		model.setNativeName(element.getNativeName());
+		// TODO align fields.isEmpty() with other DAOs on a major update - this is breaking at the moment.
+		if (fields.isEmpty() || fields.has("uuid")) {
```

Comment:
The check for fields.isEmpty() is not necessary, because fields.has(...) will return true if no fields restriction has been set anyways.